src/store/ChartState.js

```diff
@@ -152,6 +171,60 @@ class ChartState {
             this.stxx.draw();
         }
     }
+
+    ClearChart() {
+        if (!this.clearChart) return;
+        // Remove comparsions
+        for (const field in this.stxx.chart.series) {
+            this.stxx.removeSeries(field);
+        }
+        // Remove indiactors
+        for (const id in this.stxx.layout.studies) {
+            const sd = this.stxx.layout.studies[id];
+            CIQ.Studies.removeStudy(this.stxx, sd);
+        }
+        this.stxx.clearDrawings();
+        this.mainStore.crosshair.setCrosshairState(0);
+
+        this.mainStore.chart.changeSymbol(this.stxx.chart.symbol, 0);
+        this.mainStore.chartType.setType('mountain');
+    }
+
+    ImportLayout() {
+        if (!this.importLayout) return;
+        this.stxx.importLayout(this.importLayout, {
```

Comment:
@MahboobehMohammadi What happens if the developer passes an empty object by mistake?

Reply:
@morteza-binary i dont think developer with send an empty obj! it's either null or the obj that he exported!!

Reply:
@MahboobehMohammadi Anyway, I'd be better to handle race conditions.